test/hotspot/jtreg/compiler/lib/ir_framework/IRNode.java

```diff
@@ -210,13 +210,13 @@ public class IRNode {
 
```

Comment:
Suggestion: I think it will be safer to update regex for `ALLOC` and `ALLOC_OF` adding matching for RISC-V `mv` at the same time.

Reply:
> Suggestion: I think it will be safer and more consistent to update regex for `ALLOC` and `ALLOC_OF` adding matching for RISC-V `mv` at the same time.

Thanks for your review, Fixed.